abi/aapcs64/aapcs64.rst

```diff
@@ -43,6 +44,9 @@ Please report defects in this specification to the `issue tracker page
 on GitHub
 <https://github.com/ARM-software/software-standards/issues>`_.
 
+.. raw:: pdf
+
+   PageBreak
```

Comment:
Are the page breaks needed? The headers are numbered, does it really matter if we don't start chapters on new pages?

Reply:
Maybe it's my years on years familiarity with dead-tree formats, or maybe I just feel a strong need to adhere to printed text layout, but In my opinion page breaks make a pdf a lot cleaner. It gives you a mental break of sorts.

Reply:
I'd like to see a version without the page breaks before deciding :)

Reply:
GitHub seems broken. I can't seem to add the pdf on Windows or on Linux. I guess I'll have to send you it over email for now.

I must say, after re-evaluating, I'm still not a fan at all of no page breaks in pdfs.

Reply:
Yep, so, if you are not a fan at all of no page breaks, you get my blessing :)